bin/vagrant_provision.sh

```diff
@@ -69,7 +69,6 @@ VENV=/home/vagrant/virtualenv
 sudo -H -u vagrant -s -- <<EOF
 virtualenv $VENV
 source $VENV/bin/activate
-pip install -U "pip<7"
```

Comment:
I think a specific version of pip should be mentioned here. So in future it wil not be broken if peep does not work with the future versions.
``pip install -U pip==7.1.2`` should be a better choice